tests/unit/htdocs/class/CriteriaTest.php

```
@@ -0,0 +1,1829 @@
+    {
+        // Simulate: WHERE (uid > 0 AND status = 'active') AND (name LIKE '%john%' OR email LIKE '%john%')
+        $authCompo = new CriteriaCompo(new Criteria('uid', 0, '>'));
+        $authCompo->add(new Criteria('status', 'active'));
+
+        $searchCompo = new CriteriaCompo(new Criteria('name', '%john%', 'LIKE'));
+        $searchCompo->add(new Criteria('email', '%john%', 'LIKE'), 'OR');
+
+        $main = new CriteriaCompo($authCompo);
+        $main->add($searchCompo);
+
+        $expected = "(((`uid` > 0 AND `status` = 'active')) AND ((`name` LIKE '%john%' OR `email` LIKE '%john%')))";
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$expected'. (undefined)

(UnusedLocalVariable)